src/ontology/uberon-edit.obo

```diff
@@ -224330,6 +224330,20 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-03T14:38:17Z xsd:dateTime
 
+[Term]
+id: UBERON:8480035
+name: transformation zone of the cervix
+def: "Site of transition between the stratified squamous epithelium of the ectocervix and the mucus-secreting simple columnar epithelium of the endocervix. It is adjacent to the squamocolumnar junction. Before puberty and after menopause the transformation zone resides in the cervical canal. During the reproductive age of the woman is located just outside the external os. Squamous cervical cancer originates in the transition zone. Glandular cervical cancer originates in either the transformation zone or the glandular epithelium above the transformation zone." [https://www.ncbi.nlm.nih.gov/books/NBK568392/, ISBN:0781750563]
+synonym: "cervical transformation zone" EXACT [PMID:29944714]
+synonym: "cervical transformation zone epithelium" EXACT [https://myhealth.alberta.ca/Health/Pages/conditions.aspx?hwid=stt11635]
```

Comment:
Consider making this term the class label to be consistent with other similar class labels.